bin/cml.js

```diff
@@ -12,12 +12,20 @@ const { jitsuEventPayload, send } = require('../src/analytics');
 
 const aliasLegacyEnvironmentVariables = () => {
   const legacyEnvironmentPrefixes = {
-    CML_CI: 'CML_REPO',
-    CML_PUBLISH: 'CML_ASSET',
-    CML_RERUN_WORKFLOW: 'CML_WORKFLOW',
-    CML_SEND_COMMENT: 'CML_COMMENT',
-    CML_SEND_GITHUB_CHECK: 'CML_CHECK',
-    CML_TENSORBOARD_DEV: 'CML_TENSORBOARD'
+    CML_CI: 'CML',
+    CML_PUBLISH: 'CML',
+    CML_RERUN_WORKFLOW: 'CML',
+    CML_SEND_COMMENT: 'CML',
+    CML_SEND_GITHUB_CHECK: 'CML',
+    CML_TENSORBOARD_DEV: 'CML',
+    CML_ASSET: 'CML',
+    CML_CHECK: 'CML',
+    CML_COMMENT: 'CML',
+    CML_PR: 'CML',
+    CML_REPO: 'CML',
+    CML_RUNNER: 'CML',
+    CML_TENSORBOARD: 'CML',
+    CML_WORKFLOW: 'CML'
```

Comment:
what's this about?

Reply:
is this synonymous? can you provide a brief explainer that's not linking to a vague comment?

Reply:
yes would like an inline code comment here :)

Reply:
ping @0x2b3bfa0

Reply:
I'd rather not write a paper on the behavior of `yargs` environment variable parsing, and less yet amidst a code block. Resolved https://github.com/iterative/cml/pull/1272/commits/408aa3acb3150afe3c9ef5a2a185ccf8305c4230 unless proven otherwise.